user-story-nfdi4ing-instrument/functions.py

```diff
@@ -53,27 +54,38 @@ def get_metadata_display(doi):
 
     return data
 
-def format_citations(events, include_authors=False):
+def format_citations(events, doi, include_authors=False):
 
     ids = extract_ids(events)
 
     query_params = {
         "instrumentIds" : ids
     }
 
-    query = gql("""query getInstruments($instrumentIds: [String!])
-    {
-    works(ids: $instrumentIds) {
-        # authors{
-        #     title
-        #     id
-        # }
-        nodes{
-            formattedCitation
+    if include_authors:
```

Comment:
Maybe it would look cleaner if we use a variable replacement for that section like we do with $intrumenIds. Have you consider that option. @bklaing2